tests/lib/rules/jsx-no-literals.js

```diff
@@ -128,37 +128,42 @@ ruleTester.run('jsx-no-literals', rule, {
       parser: parsers.BABEL_ESLINT
     }, {
       code: `
-        <Foo bar='test'>
+        <Foo bar={'test'}>
```

Comment:
changing any "valid" tests makes this a breaking change. i think this needs to be behind an option that's disabled by default, so that this PR does not touch any "valid" tests, only adds ones (and invalid ones too).